src/shared/features/AppletSettings/ReportConfigSetting/ReportConfigSetting.tsx

```diff
@@ -54,15 +63,12 @@ import { useCheckReportServer, useDefaultValues } from './ReportConfigSetting.ho
 import { usePrompt } from '../AppletSettings.hooks';
 import { REPORT_SERVER_INSTRUCTIONS_LINK } from './ReportConfigSetting.const';
 
-export const ReportConfigSetting = ({
-  isDashboard,
-  onSubmitSuccess,
-  activity,
-  activityFlow,
-}: ReportConfigSettingProps) => {
+export const ReportConfigSetting = ({ isDashboard, onSubmitSuccess }: ReportConfigSettingProps) => {
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
   const navigate = useNavigate();
+  const { activity } = useCurrentActivity();
```

Comment:
have you checked on the Dashboard Applet settings?
it could be not working

Reply:
it works fine

Reply:
now